app/controllers/schedule_controller.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class ScheduleController < ApplicationController
+  def save
+    @document = Document.with_current_edition.find_by_param(params[:id])
+    @edition = @document.current_edition
+    current_revision = @edition.revision
+    date_issues = Requirements::DatetimeChecker.new(date_params).date_issues
+    datetime_issues = Requirements::DatetimeChecker.new(datetime_params).datetime_issues
+
+    if date_issues.any? || datetime_issues.any?
+      flash[:day_issues] = date_issues.items_for(:scheduled_day)
```

Comment:
We could assign `@issues` here and do the extraction in the view - I think that's how the other views do it.

Reply:
Opting for Kevin's suggestion of using a redirect + flash below 👇 n